File: solutions/c/darts/1/darts.c
#include "darts.h"
#include "math.h"
int score (coordinate_t landing_position) {
    float dist_from_center = sqrt(landing_position.x * landing_position.x + landing_position.y * landing_position.y);

    if(dist_from_center <= 1) {
        return 10;
    }
    if(dist_from_center <= 5) {
        return 5;
    }
    if(dist_from_center <= 10) {
        return 1;
    }
    return 0;
}

File: solutions/java/darts/1/src/main/java/Darts.java
import java.lang.Math;

class Darts {
    // (x)² + (y)² < radius²

    int score(double xOfDart, double yOfDart) {
        // inner circle
        if (Math.pow(xOfDart, 2) + Math.pow(yOfDart, 2) < Math.pow(1, 2)) {
            return 10;
        }
        if (Math.pow(xOfDart, 2) + Math.pow(yOfDart, 2) < Math.pow(5, 2)) {
            return 5;
        }
        if (Math.pow(xOfDart, 2) + Math.pow(yOfDart, 2) < Math.pow(10, 2)) {
            return 1;
        }
        return 0;
    }


}
